index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <img src="/images/Welcome.png" alt="welcome button at the top left">
```

Comment:
- [ ]  it would be better if you change this to the `<a>` tag because the user when clicking on the logo should navigate to the home page and also improve user experience so please try to do that

Reply:
should I remove the header? and replace it with the < a> tag? or i should just nest the img tag in an <a> tag